02-when-vector-data-dont-line-up-CRS.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: "Lesson 02: When Vector Data Don't Line Up - Handling Spatial Projections 
-- CRS - in R"
+title: "Lesson 02: When Vector Data Don't Line Up -- Handling Spatial
```

Comment:
This is a can of worms. You can talk to Leslie but do so with caution. We don't have a full style guide and our science writer (Kendra) is gone :( .